Add required make target "vhdl.install-plugin".

Index: ghdl-0.30~svn20130213/ghdl/translate/gcc/Make-lang.in
===================================================================
--- ghdl-0.30~svn20130213.orig/ghdl/translate/gcc/Make-lang.in	2012-12-11 03:46:11.000000000 +0100
+++ ghdl-0.30~svn20130213/ghdl/translate/gcc/Make-lang.in	2013-12-08 16:22:44.634533000 +0100
@@ -179,6 +179,8 @@
 # Install the library
 	$(MAKE_IN_VHDL) install-ghdllib
 
+vhdl.install-plugin:
+
 install-info:: $(DESTDIR)$(infodir)/ghdl.info
 
 vhdl.install-info: doc/ghdl.info
